Python/ExcelOutputClass.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+"""
+Spyder Editor
+
+This is a temporary script file.
+"""
+
+import os
+import xlsxwriter
+import openpyxl
+
+os.chdir(r'\Users\Ben\Documents\GitHub\Ben-Projects')
+
+def make_excel_file(title):
```

Comment:
That last code was stupid, I forgot you were passing the file name, I left that out.
![image](https://cloud.githubusercontent.com/assets/9099993/23808143/f8fb05ec-0585-11e7-947c-5992ce12363f.png)